model/src/main/scala/uk/gov/ons/addressIndex/model/AddressIndexModel.scala

```diff
@@ -16,11 +15,11 @@ case class AddressIndexSearchRequest(
 )
 
 sealed trait AddressScheme {
-  override def toString() : String
+  override def toString: String
 }
 case class PostcodeAddressFile(override val toString : String) extends AddressScheme
 case class BritishStandard7666(override val toString : String) extends AddressScheme
-case class InvalidAddressSchemeException(str : String) extends IllegalArgumentException(s"Invalid address scheme $str")
+case class UnsupportedScheme(override val toString : String) extends AddressScheme
```

Comment:
why? This doesnt make any sence. How can an `UnsupportedScheme` be a type of `AddressScheme`